calendar-lite.js

```diff
@@ -677,6 +677,14 @@ class CalendarLite extends GestureEventListeners(PolymerElement) {
 
   _show(e) {
     this.current_page = e.target.attributes.type.value;
+    setTimeout(() =>{
+      let list = this.shadowRoot.querySelectorAll('.yearItem');
+      Array.prototype.forEach.call(list, (el) => {
+        let found = Array.prototype.find.call(list, el => Number(el.innerText) === this._getSelectedYear());
```

Comment:
Hard to understand this code.. my questions:
- do you really need Array.prototype...call
- is the double reiteration through the `list` needed
- the semantics of the word 'found' indicates it holds a boolean, a better naming might be 'selectedYearElem'